hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -431,6 +435,20 @@ public ObjectNamespace getObjectNamespace() {
         return definitions;
     }
 
+    public List<Index> getIndexList() {
```

Comment:
Is this method really needed? Looks like for purpose of this PR it's never invoked if global indexes are not enabled, so it's basically equivalent to the existing MapContainer#getIndexes().

Reply:
Removed the method